Products/ZenUtils/elastic/client.py

```diff
@@ -38,8 +38,27 @@ def __init__(self):
         used across all requests that the client makes.
         """
         self.session = None
+        self.cc_version = ""
+        if self._checkHothOrNewer():
```

Comment:
We should probably create a method in controlplane.py that returns the CC url depending on the cc version